### ISSUE TITLE
send statistics to not working in v2.6

### ISSUE THREAD

**Issue by StephenPCG** [grobian/carbon-c-relay#253]
Version: v2.6

Conf:
```
cluster default
    fnv1a_ch replication 1
        127.0.0.1:2103 proto tcp
    ;

send statistics to
    default
    ;

match *
    send to
        default
    stop
    ;
```

Failed to start, error:
```
$ carbon-c-relay -f /tmp/relay.conf
[2017-03-17 11:00:40] starting carbon-c-relay v2.6 (9cdf4d), pid=32949
configuration:
(snip)

[2017-03-17 11:00:40] expected cluster name after 'send statistics to'
[2017-03-17 11:00:40] failed to read configuration '/tmp/relay.conf'
(program exited, exit code: 1)
```

**Comment by grobian** [comment]
hmmm, v2.6 (9cdf4d) parses your config fine here

**Comment by StephenPCG** [comment]
Ohh, I'm sorry for the noise. I rechecked my configuration and find that the configuration is actually like this:
```
send statistics to
    default;
```

There was no blank between cluster name ``default`` and ``;`` . I didn't remember how the conf content pasted here is not the same as the original file 😂 

I'm really sorry for the noise, and thanks very much for quickly fixing #252 , I think I can switch back to use master branch now.

**Comment by StephenPCG** [comment]
Well, I think I find out why I pasted a different config content. Originally, I wrote the ``send statistics to`` clause in one line:

```
send statistics to default;
``` 

And later changed to
```
send statistics to
    default;
```

And when pasting here, I somehow changed the semicolon position.

I'm not sure if this should be treated as a bug, I just checked ``cluster`` and ``match`` clause, and found that only ``send statistics to`` complains when there is not space before the semicolon.

```
cluster default
    fnv1a_ch replication 1
        127.0.0.1:2103 proto tcp;

send statistics to default
    ;

match * send to default stop;
```

The above config parsed fine.
